lib/playground.dart

```diff
@@ -193,27 +194,23 @@ class Playground {
     _editpanel.children.first.attributes['flex'] = '';
     editor.resize();
 
-    keys.bind('ctrl-s', _handleSave);
-    keys.bind('ctrl-enter', _handleRun);
-    keys.bind('f1', () {
+    keys.bind(['ctrl-s'], _handleSave);
+    keys.bind(['ctrl-enter'], _handleRun);
+    keys.bind(['f1'], () {
       _toggleDocTab();
       _handleHelp();
     });
+
+    keys.bind(['crtl-space', 'macctrl-space'], (){
+      editor.completionAutoInvoked = false;
+      editor.execCommand('autocomplete');
+      new Timer(const Duration(milliseconds: 500), _handleHelp);
```

Comment:
Let's remove this for now; there's no good way to get a signal that the codemirror completion popup became visible. The user can cursor around the popup in order to see the doc view update if they like.